synctl/cli.py

```diff
@@ -502,6 +512,92 @@ def pop_size_estimate(self):
+
+    def cost_estimate(self):
+        print("Please answer below questions for estimating the cost of Synthetic tests running on Instana hosted PoPs\n ")
+        try:
+            while True:
+                locations = int(self.ask_question("How many managed locations will be used? "))
+                if locations > 0:
+                    while True:
+                        api_simple = int(self.ask_question("How many API Simple tests do you want to create? (0 if no) "))
+                        if api_simple > 0:
+                            while True:
+                                api_simple_frequency = int(self.ask_question("What is the test frequency for your API Simple tests? (1-120)  "))
+                                if api_simple_frequency > 0 and api_simple_frequency <= 120:
+                                    api_simple_test_exec = self.test_exec_estimate(api_simple, api_simple_frequency, locations)
+                                    # resource units per month
+                                    api_simple_res = api_simple_test_exec * self.factors["APISimple"]
+                                    break
+                                else:
+                                    print("frequency is not valid, it should be in [1,120]")
+                            break
+                        elif api_simple == 0:
+                            api_simple_test_exec = 0
+                            api_simple_res = 0
+                            break
+                        else:
+                            print("Invalid input")
+                    while True:
+                        api_script = int(self.ask_question("How many API Script tests do you want to create? (0 if no) "))
+                        if api_script > 0:
+                            while True:
+                                api_script_frequency = int(self.ask_question("What is the test frequency for your API Script tests? (1-120) "))
+                                if api_script_frequency > 0 and api_script_frequency <= 120:
+                                    api_script_test_exec = self.test_exec_estimate(api_script, api_script_frequency, locations)
+                                    api_script_res = api_script_test_exec * self.factors["APIScript"]
+                                    break
+                                else:
+                                    print("frequency is not valid, it should be in [1,120]")
+                            break
+                        elif api_script == 0:
+                            api_script_test_exec = 0
+                            api_script_res = 0
+                            break
+                        else:
+                            print("Invalid input")
+
+                    while True:
+                        browser_script = int(self.ask_question("How many Browser tests (Webpage Action, Webpage Script and BrowserScript) do you want to create? (0 if no) "))
```

Comment:
can we simplify this question to? @rui1234 FYI.
```suggestion
                        browser_script = int(self.ask_question("How many Webpage Action, Webpage Script and BrowserScript tests do you want to create? (0 if no) "))
```
```
How many Webpage Action, Webpage Script and BrowserScript test do you want to create? (0 if no)
```